build/Build.cs

```diff
@@ -0,0 +1,107 @@
+// ReSharper disable RedundantUsingDirective - prevent PrettyBot from getting confused about unused code.
```

Comment:
This looks useful - if that's all we need to make NUKE and R# play nicely together then that's a good win.

Reply:
Yup - this is the only thing that we need to worry about for the prettybot script. This has been running "in the real world" in OctopusCLI and appears to be working well :success: